### ISSUE TITLE
HuskySort using Collator doesn't sort in correct order

### ISSUE THREAD

**Issue by NiftyMule** [rchillyard/HuskySort#53]
HuskySort using Collator doesn't use Collator.compare() in second pass when fixing remaining insertionSort

**Comment by rchillyard** [comment]
Yes, HuskySort sorts Chinese in Unicode order which, at the time, seemed appropriate. Part of the rationale for the Fall 2021 Team Project (INFO 6205) was to figure out exactly how it should be done properly.
This is being fixed in the branch
